src/pages/Confirm/index.jsx

```diff
@@ -1,3 +1,5 @@
+// @ts-nocheck
```

Comment:
Can you add a screenshot and description of the error on this place, please?

Reply:
![image](https://user-images.githubusercontent.com/61360270/142059256-d05e13fa-3e00-494a-bc3c-048341b68aea.png)
![image](https://user-images.githubusercontent.com/61360270/142059261-622c716f-2faf-481d-97d7-9e6a3863c260.png)

Reply:
In the last screenshot, the problem will not be that the function "submitRating" is not expecting any param, and handleClick event is trying to pass Event Param?

And in the first screenshots it seems that the component Text does not have defined the types of attributes that it expects

@Israel-Laguan @e-azocar